index.html

```diff
@@ -0,0 +1,357 @@
+            <div><i class="fa-sharp fa-solid fa-bell"></i></div>
+            <div><i class="fa-solid fa-user"></i></div>
+        </div>
+    </div>
+
+
+
+    <div class="container">
+        <div class="video-box">
+            <video width="100%" height="600" controls>
+                <source src="movie.mp4" type="video/mp4">
+                <source src="movie.ogg" type="video/ogg">
+                Your browser does not support the video tag.
+              </video>
+       
+
+        <div class="about-video">
+            <div class="video-title"><p>Contributing to Open Source Part I: The Easy Way</p></div>
+            <div class="controls">
+                <div class="chanel">
+                    <i class="fa-solid fa-user"></i>
+                    <p>The Odin Project <br> <span>1.11K subscribers</span></p>
+                    <button type="submit" class="sub">Subscribe</button>
+                </div>
+                
+                
+              <div class="icons">
+                <div class="like-dislike">
+                    <button type="submit" class="like"><i class="fa-solid fa-thumbs-up"></i><span>488</span></button>
+                    <button type="submit" class="dislike"><i class="fa-solid fa-thumbs-down"></i></button>
+                </div>
+                <button type="submit" class="share"><i class="fa-solid fa-share"></i><span>Share</span></button>
+                <button type="submit" class="clip"><i class="fa-solid fa-scissors"></i><span>Clip</span></button>
+                <button type="submit" class="save"><i class="fa-solid fa-floppy-disk"></i><span>Save</span></button>
+                <button type="submit" class="more"><i class="fa-solid fa-ellipsis"></i></button>
+            </div>
+            </div>
+        </div>
+
+        <div class="discription">
+            <p>68K views  8 years ago</p>
+            <p>How do you contribute to open source?  In this video, I'll describe the easiest way to make a simple contribution to an open source project
+                 -- without using the command line, a text editor, "forking a repo", or even touching Git outside of Github. 
+                  I'll also describe what kinds of contributions open source software projects are typically looking for. 
+                   As an example, I use submitting a student solution to a web development exercise from The Odin Project, a free online curriculum 
+                for learning web development with Ruby on Rails.  You can find more information about that project at http://theodinproject.com.
+            </p>
+
+        </div>
+
+        <div class="comment-sort">
+            <div>17 Comments</div>
+            <div><i class="fa-solid fa-arrow-down-short-wide"></i> <span>Sort by</span></div>
+        </div>
+
+        <div class="comment-input">
+            <span><i class="fa-solid fa-user"></i></span><input type="text" placeholder="Add a comment..." id="comment-place">
+           
+        </div>
```

Comment:
Please consider that you could be more clear and direct with this point. Which semantic HTML tags do you mean? Where should the student put them? Also, consider giving a link about the topic or giving a code example: 
```html 
<header> 
    <nav> 
        nav links 
    </nav> 
</header> 
``` 

Kindly remember that using semantic elements also improves the code structure + accessibility of the website. You can [learn more here](https://www.simplilearn.com/tutorials/html-tutorial/html-semantics#:~:text=The%20HTML%20semantics%20refers%20to,and%20layout%20of%20web%20pages.) 😄.

Reply:
As the previous reviewer requested, please be more precise and specific about where you referring and what solution you're recommending. Right now, you selected all the lines from line 60 to line 83 and where you are pointing is pretty general and vague. Also please check if you are giving the correct link because you are asking for HTML semantics and I guess it would be better to use [HTML Semantic](https://www.w3schools.com/html/html5_semantic_elements.asp) topic instead of HTML tags. 👌